samples/file-encryptor/README.md

```diff
@@ -1,27 +1,37 @@
 # The File-Encryptor Sample
 
-OE SDK comes with a default crypto support library that supports a [subset of the open sources mbedTLS](https://github.com/openenclave/openenclave/blob/master/docs/MbedtlsSupport.md) library.
-This sample demonstrates how to perform simple file cryptographic operations inside an enclave using mbedTLS library.
+OE SDK comes with a default crypto support libraries that supports a [subset of the open sources mbedTLS](https://github.com/openenclave/openenclave/blob/master/docs/MbedtlsSupport.md) library and [subset of the open sources OpenSSL](https://github.com/openenclave/openenclave/blob/master/docs/OpenSSLSupport.md)
+library. This sample demonstrates how to perform simple file cryptographic operations inside an enclave either using mbedTLS library or OpenSSL library. Developers can use cryptographic operations from both the libraries at the same time but this sample application only demonstrates the use of either mbedTLS or OpenSSL library at each time. The crypto support library for sample application is selected using build variable `OE_CRYPTO_LIB` in [Makefile](https://github.com/openenclave/openenclave/blob/master/samples/file-encryptor/Makefile) or [CMakeList.txt](https://github.com/openenclave/openenclave/blob/master/samples/file-encryptor/CMakeLists.txt) file. For OpenSSL library `OE_CRYPTO_LIB` should be set to `openssl` and for mbedTLS library it should set to `mbedtls`.
```

Comment:
```suggestion
library. This sample demonstrates how to perform simple file cryptographic operations inside an enclave using either the mbedTLS library or the OpenSSL library. Developers can use cryptographic operations from both the libraries at the same time but this sample application only demonstrates the use of either mbedTLS or OpenSSL library at each time. The crypto support library for sample application is selected using a build variable `OE_CRYPTO_LIB` in [Makefile](https://github.com/openenclave/openenclave/blob/master/samples/file-encryptor/Makefile) or [CMakeList.txt](https://github.com/openenclave/openenclave/blob/master/samples/file-encryptor/CMakeLists.txt) file. To use the OpenSSL library, `OE_CRYPTO_LIB` should be set to `openssl` and for mbedTLS library it should set to `mbedtls`.
```

Also please split into multiple source lines since you can see that GitHub doesn't deal well with showing the diffs in my suggestion with such long lines.

Reply:
Consider either pointing out that the `OE_CRYPTO_LIB` value is case-sensitive (it's used as a path segment in the build, which is case-sensitive on Linux, CMake `STREQUAL` is also case-sensitive) or updating the sample build files so that it is not (e.g. force lower-case the value before using it).